rss/medianamespace.cpp

```diff
@@ -27,24 +27,47 @@ void parse_media_node(xmlNode* node, Item& it)
 		Enclosure {
 			get_prop(node, "url"),
 			get_prop(node, "type"),
+			"",
+			"",
 		}
 		);
 		for (xmlNode* mnode = node->children; mnode != nullptr; mnode = mnode->next) {
-			parse_media_node(mnode, it);
+			parse_media_node(mnode, it, &it.enclosures.back());
 		}
 	} else if (node_is(node, "description", MEDIA_RSS_URI)) {
+		const std::string description = get_content(node);
 		const std::string type = get_prop(node, "type");
+		const std::string mime_type = (type == "html" ? "text/html" : "text/plain");
 		if (it.description.empty()) {
-			it.description = get_content(node);
-			if (type == "html") {
-				it.description_mime_type = "text/html";
-			} else {
-				it.description_mime_type = "text/plain";
+			it.description = description;
+			it.description_mime_type = mime_type;
+		}
+		if (enclosure) {
+			enclosure->description = description;
+			enclosure->description_mime_type = mime_type;
+		} else {
+			if (it.description.empty()) {
```

Comment:
I guess this block can be removed, given that the description is already set above